### ISSUE TITLE
Size of w2 in target classificatin module

### ISSUE THREAD

**Issue by shuida** [visionml/pytracking#57]
@martin-danelljan 
Thanks for your algorithm and implementation, both of which are perfect! However, I have been bothering by a question for a long time. In pytracking/parameter/atom/default.py, the default size of w2 in target classification module is 4x4. Why did you choose the even size rather than odd size such as 3x3 or 5x5？ Doesn't even size lead to inaccurate locating? What's its advantages of 4x4 over 3x3 or 5x5？

**Comment by martin-danelljan** [comment]
Hi. Thanks for the appreciation!

Even size does not lead to inaccurate tracking if implemented properly. That is if the offsets and centering needed for an even kernel is handled correctly. We do this in our code, which allows you to use any kernel size without a problem. When we experimented with different kernel sizes, we found 4 x 4 to be the best trade-off between speed and accuracy. Results with 3 x 3 was a bit worse. But this can depend on what kind of backbone network and feature resolution you use.
